src/styles/index.scss

```diff
@@ -51,12 +51,20 @@ button,
 	border: 1px solid black !important;
 	background: white !important;
 	font-weight: bold !important;
+	height: 30px !important;
+}
+
+.sc-gsDKAQ {
+	svg {
+		width: 18px !important; 
+		height: 18px !important; 
+		top: 3px !important; 
+	}
```

Comment:
done to make the civic button the same height as ours, other solution is to increase our button size to the size of the identity button